rfq/negotiator.go

```diff
@@ -177,9 +177,12 @@ func (n *Negotiator) HandleOutgoingBuyOrder(buyOrder BuyOrder) error {
 
 		if n.cfg.PriceOracle != nil && assetSpecifier.IsSome() {
 			// Query the price oracle for a bid price.
+			//
+			// TODO(ffranr): Add assetMaxAmt to BuyOrder and use as
```

Comment:
This TODO remains unresolved and we don't pass any amount to the oracle.

Reply:
Adding and populating that field in the `BuyOrder` would require an RPC change. I mean to do that in a future PR. I think this PR is getting too long.

Reply:
But at this specific call site, we only pass in `fn.None` for all the values. So we're not passing any amount when requesting a rate for the hint. Is that an allowed way to call into the oracle, with none of the `_max_amount` fields set?

Reply:
I think it should technically be a valid way of calling the price oracle. If upper bounds aren't given then the price would just be less accurate. But we should complete the TODO. I'll add the correct field to `BuyOrder` in a separate PR.